packages/ProgressAccordion/src/components/Indicator/Indicator.styles.js

```diff
@@ -69,7 +69,8 @@ export const indicatorStyles = css`
 `;
 
 export const indicatorDotStyles = css`
-  ${stylers.z(2)};
+  ${stylers.fontSize()}
```

Comment:
So do we need semi-colons with these or not? 😄

Reply:
Not. If the `styler` generates the whole line, then it generates the _whole_ line.  I think there's also a `stylers.fontSizeValue()` that does NOT produce the `;`... but you'd have to use it like:

```
font-size: ${stylers.fontSizeValue()};
```